packages/commerce-sdk-react/src/auth/index.ts

```diff
@@ -22,6 +22,7 @@ type Helpers = typeof helpers
 interface AuthConfig extends ApiClientConfigParams {
     redirectURI: string
     proxy: string
+    slasProxy: string
```

Comment:
We are setting up a new proxy in ssr.js to insert the client secret. This new slasProxy is how I am consuming that separate proxy at this time